R/impute_daily_incidence.R

```diff
@@ -0,0 +1,23 @@
+#' Imputed incidence value for the single day with reporting anomaly
+#'
+#' @param data time series data for one location in adjustment case
+#' @param adjustment_case a case to adjust
+#' @param measure case or death
+#' @return incidence columns after adjustments
+#' @export
+
+impute_daily_incidence <- function(data, adjustment_case, measure) {
+  if (measure == "deaths") {
+    imputed_data <- covidData::jhu_deaths_imputed_data
+  } else {
+    imputed_data <- covidData::jhu_cases_imputed_data
+  }
+
+  location <- unique(data$location)
+  date <- as.Date(adjustment_case$date)
+  
+  unique(imputed_data[which(
```

Comment:
Why is there a call to `unique` here?

Reply:
This is not a bug, but let's remove `unique` from here and instead ensure that at the time we generate imputed values we save only a single imputed value for each combination of location and date. It seems like we could do this by inserting a check of whether the combination of `fips` and `adjustment_date` is already in the `results` data frame at https://github.com/reichlab/covidData/blob/fill-nas/code/data-processing/get_imputed_data.R#L125